plugins/module_utils/ilo_redfish_utils.py

```diff
@@ -51,6 +99,423 @@ def get_ilo_sessions(self):
         result["ret"] = True
         return result
 
+    def get_systems_data(self):
```

Comment:
Getting system info is standard Redfish functionality; we have this today as "get_system_inventory". Why is this needed?

Reply:
This function is being used in every subsequent function in the file to retrieve the entire systems data. **"get_system_inventory"** returns only certain attributes, whereas we need many different attributes in many of the function.
Please refer get_service_bios_attributes() for this.

Reply:
I would much rather enhance "get_system_inventory" in the standard module (or even have a different method in the standard module) to retrieve all properties in a desired system. This is not something iLO-specific.

Reply:
Hi, we noticed that this seems to be an unnecessary function. An oversight on our part, referred **redfish_utils** and will be following that as an example. Sorry for the confusion.

Reply:
Resolving this conversation, thanks!